docs/source/markdown/podman-pod-clone.1.md.in

```diff
@@ -99,22 +99,27 @@ clone process has completed. All containers within the pod are started.
 @@option volumes-from
 
 ## EXAMPLES
+
+Clone the pod pod-name to a new pod.
```

Comment:
```suggestion
Clone the pod `pod-name` to a new pod.
```

When referring to a particular instance of some thing, it might be best to backtick it.